https://issues.redhat.com/browse/ACM-11617 (#6528)
open-cluster-management/rhacm-docs: release_notes/whats_new.adoc

* You can customize the Observatorium API and Alertmanager URLs. See link:../observability/customize_observability.adoc#custom-obervatorium-alert-url[Customizing the managed cluster Observatorium API and Alertmanager URLs].

> * You can enable the _Fleet_ switch to view the update risks that are present for your managed clusters. See, link:../observability/insights_intro.adoc#update-risks[Viewing update risk predictions]. 

Author: ```suggestion
* You can enable the _Fleet_ switch to view the potential risks for updating your managed clusters. See, link:../observability/insights_intro.adoc#update-risks[Viewing update risk predictions]. 
```

> * You can view the update risks that are present for your managed clusters by enabling the _Fleet_ switch. See, link:../observability/insights_intro.adoc#update-risks[Viewing update risk predictions]. 

Reviewer: `You can now enable the _Fleet_ switch to view update risk predictions for your managed cluster.`

The only other thing is I would have liked to see why I want to do this in the intro and what's new. What risks? Just another line/word ortwo about the benefit of this function, especially in the What's new.

> * You can view the update risks that are present for your managed clusters by enabling the _Fleet_ switch. See, link:../observability/insights_intro.adoc#update-risks[Viewing update risk predictions]. 

Author: ```suggestion
* You can enable the _Fleet_ switch to view the update risks that are present for your managed clusters. See, link:../observability/insights_intro.adoc#update-risks[Viewing update risk predictions]. 
```
